RoosterPlanner.Data/Common/UnitOfWork.cs

```diff
@@ -1,10 +1,7 @@
 ﻿using System;
```

Comment:
Deze changes in deze file lijken mij niet relevant voor de PR. De volgende keer zou ik dit opsplitsen in een aparte feature branch.